pygfx/utils/load.py

```diff
@@ -41,36 +41,60 @@ def load_meshes(path):
     ----------
     path : str
         The location where the mesh or scene is stored.
+        Can be a local file path or a URL.
 
     Returns
     -------
     meshes : list
         A list of loaded meshes.
 
     """
-
     import trimesh  # noqa
 
-    scene = trimesh.load(path)
+    # Trimesh's load() performs a similar check and refers
+    # loading from URLs to load_remote()
```

Comment:
If that's the case why can't we just use `trimesh.load`?

Reply:
When `trimesh.load` detects a URL it raises a `ValueError` telling us to use `trimesh.load_remote`:

```python
>>> scene = tm.load("https://github.com/mikedh/trimesh/raw/main/models/cube_blender_uv.ply")
...
ValueError: use load_remote to load URL: https://github.com/mikedh/trimesh/raw/main/models/cube_blender_uv.ply
```

According to the docstring of `tm.load_remote` the logic is to avoid accidental network requests:

```
Signature: tm.load_remote(url, **kwargs)
Docstring:
Load a mesh at a remote URL into a local trimesh object.

This must be called explicitly rather than automatically
from trimesh.load to ensure users don't accidentally make
network requests.
```

I thought the convenience of having one function do both was preferable but I see their point. Any preference?

Reply:
How about you add a keyword argument `network_ok=False` to force users to opt-in to the magic behaviour :)

Reply:
Added via ed99a85